### PR TITLE
ldapadmin - Add webservice for org type distribution

### PR DESCRIPTION
**Pull request by Vampouille**

### CODE REVIEW

**Comment by Vampouille** [comment]
@pmauduit can you review this PR ? I can rebase it if you want ?

File: ldapadmin/src/main/java/org/georchestra/ldapadmin/ws/backoffice/orgs/OrgsController.java [hunk 1]
```diff
@@ -332,20 +274,72 @@ public void getUserRequiredFields(HttpServletResponse response) throws IOExcepti
...
-            JSONArray fields = new JSONArray();
-            for(String field : this.orgDao.getOrgTypeValues())
-                fields.put(field);
-            ResponseUtil.buildResponse(response, fields.toString(4), HttpServletResponse.SC_OK);
-        } catch (Exception e) {
-            LOG.error(e.getMessage());
-            ResponseUtil.buildResponse(response, ResponseUtil.buildResponseMessage(false, e.getMessage()),
-                    HttpServletResponse.SC_INTERNAL_SERVER_ERROR);
-            throw new IOException(e);
+    public void getOrgTypeVAlues(HttpServletResponse response) throws IOException, JSONException {
```
Comment (pmauduit) [inline ldapadmin/src/main/java/org/georchestra/ldapadmin/ws/backoffice/orgs/OrgsController.java]: Values (without capital A) ?

**Comment by Vampouille** [comment]
@pmauduit I just fix typo on `getOrgTypeVAlues()` and rebase PR on master. Can you take a look at result ?

**Comment by pmauduit** [comment]
Looks good

**Pull request merged**